doc/coq.rst

```diff
@@ -62,7 +62,8 @@ stanza:
 
 The stanza builds all the ``.v`` files in the given directory and its
 subdirectories if the :ref:`include-subdirs <include-subdirs-coq>` stanza is
-present.
+present. The dependencies between the ``.v`` files are computed using
+``coqdep``, which is called once per theory and produces a ``.theory.d`` file.
```

Comment:
"between" / "among .v files" ?

I'm not sure this implementation detail is relevant here to be honest. If you want to have it IMHO it needs to be moved out to a section "Computing dependencies", also the produced file should be fully named.

Reply:
We had nothing talking about how dependencies are computed, but I thought it would be worth mentioning the `.theory.d` here.

Reply:
I mean, we should say the file we produce is named "$theory_name.theory.d"

But IMHO that's an implementation detail.

Reply:
> We had nothing talking about how dependencies are computed, but I thought it would be worth mentioning the `.theory.d` here.

Ok sounds good, but IMHO let's put then that in its own section, and explain where the files are placed, and that they are in makefile format (and that we parse that)